### PR TITLE
SUL23-434: Set up shared tags events view

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
Since this almost worked as expected using the Shared Tags view, I took the Shared Tags view, duplicated it, and then added additional criteria to ensure that it was picking events content only, and the event end has not yet passed. I also changed the list paragraph config to allow the new view to be allowed for use in lists.

This will make it so the only change that will be required for the Branner example will be to change the view in use.

# Review By (Date)
- When does this need to be reviewed by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: docroot/profiles/lagunita/sul_profile/config/sync/views.view.sul_shared_tag_events.yml [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+    cache_metadata:
+      max-age: -1
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  card_grid:
```
Comment (pookmish) [inline docroot/profiles/lagunita/sul_profile/config/sync/views.view.sul_shared_tag_events.yml:369]: You will want to create a "graphql" display that matches all the filters. Make sure you set the pager setting to match other graphql displays. You can see those in the original event view.

**Review by pookmish — CHANGES_REQUESTED** [review]
Needs gql display

File: docroot/profiles/lagunita/sul_profile/config/sync/views.view.sul_shared_tag_events.yml (hunk 1 above)
Comment (imonroe) [reply docroot/profiles/lagunita/sul_profile/config/sync/views.view.sul_shared_tag_events.yml:369]: graphql display added.

**Review by pookmish — APPROVED** [review]
LGTM

**Pull request merged**